{
  "error_message": "Route %s used \"headers\" inside \"unstable_after(...)\". This is not supported. If you need this data inside an \"unstable_after\" callback, use \"headers\" outside of the callback. See more info here: https://nextjs.org/docs/canary/app/api-reference/functions/unstable_after",
  "file_path": "packages/next/src/server/request/headers.ts",
  "occurrence_count": 1
}
